src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Blur.Android.cs

```diff
@@ -0,0 +1,205 @@
+﻿/*
+
+Implementation based on https://github.com/roubachof/Sharpnado.MaterialFrame.
+with some modifications and removal of unused features.
+
+*/
+
+using System;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Views;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.UI;
+using Uno.UI.Controls;
+using Uno.UI.Xaml.Media;
+using Windows.UI.Xaml.Controls;
+
+namespace Windows.UI.Xaml.Media
+{
+	public partial class AcrylicBrush
+	{
+		private static bool? _isStockBlurSupported = null;
+
+		private const float AndroidBlurRadius = 20;
+		private const float StyledBlurRadius = 64;
+
+		internal const int BlurAutoUpdateDelayMilliseconds = 100;
+		internal const int BlurProcessingDelayMilliseconds = 10;
+
+		/// <summary>
+		/// If set to <see langword="true"/>, the rendering result could be better (clearer blur not mixing front elements).
+		/// However due to a bug in the Xamarin framework https://github.com/xamarin/xamarin-android/issues/4548, debugging is impossible with this mode (causes SIGSEGV).
+		/// A suggestion would be to set it to false for debug, and to true for releases.
+		/// </summary>
+		public static bool ThrowStopExceptionOnDraw { get; set; } = false;
+
+		private bool IsAndroidBlurPropertySet => AndroidBlurRadius > 0;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)